app/chewy/statuses_index.rb

```diff
@@ -50,7 +50,7 @@ class StatusesIndex < Chewy::Index
     root date_detection: false do
       field :account_id, type: 'long'
 
-      field :text, type: 'text', value: ->(status) { [status.spoiler_text, Formatter.instance.plaintext(status)].join("\n\n") } do
+      field :text, type: 'text', value: ->(status) { [status.spoiler_text, Formatter.instance.plaintext(status)].concat(status.media_attachments.map(&:description)).join("\n\n") } do
```

Comment:
I have to admit I don't remember how eager loading works in this class but you need to make sure this doesn't create n+1 queries. While the batch import is only used once, usually, the faster it completes, the better...